<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="searchIndicatorsTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>720</y>
     <width>381</width>
     <height>35</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QScrollArea" name="reccomendsScrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>381</width>
     <height>120</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="RScrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>379</width>
      <height>118</height>
     </rect>
    </property>
    <widget class="QListWidget" name="recommendsListWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-10</y>
       <width>381</width>
       <height>130</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="searchLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Поиск среди всех показателей&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="allIndicatorsLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Все показатели&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="reccomendsLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>381</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Рекомендуемые&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="allIndicatorsScrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>381</width>
     <height>284</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="AIScrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>379</width>
      <height>282</height>
     </rect>
    </property>
    <widget class="QListWidget" name="allIndicatorsListWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>381</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="lastSearchedLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Последние&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="lastSearchedScrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>381</width>
     <height>134</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="LSScrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>379</width>
      <height>132</height>
     </rect>
    </property>
    <widget class="QListWidget" name="lastSearchedListWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>381</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolButton" name="updateIndicatorsToolButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>70</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>↻</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
